src/styles/auto-prefix.js

```diff
@@ -3,61 +3,97 @@ import warning from 'warning';
 
 const prefixers = {};
 
+let hasWarnedAboutNavigator = false;
+
+function getPrefixer(userAgent) {
+  if (typeof navigator !== 'undefined') {
+    userAgent = navigator.userAgent;
```

Comment:
this might break the prefixer if someone polyfils navigator for some other library. 

```js
if (!userAgent && typeof navigator !== 'undefined') {
  userAgent = navigator.userAgent;
}
```